src/mono/mono.proj

```diff
@@ -543,6 +550,7 @@
       <_Objcopy Condition="'$(Platform)' == 'armv6'">arm-linux-$(_LinuxAbi)eabi$(_LinuxFloatAbi)-$(_Objcopy)</_Objcopy>
       <_Objcopy Condition="'$(Platform)' == 'arm64'">aarch64-linux-$(_LinuxAbi)-$(_Objcopy)</_Objcopy>
       <_Objcopy Condition="'$(Platform)' == 's390x'">s390x-linux-$(_LinuxAbi)-$(_Objcopy)</_Objcopy>
+      <_Objcopy Condition="'$(Platform)' == 'ppc64le'">ppc64le-linux-$(_LinuxAbi)-$(_Objcopy)</_Objcopy>
```

Comment:
The GNU triplet for ppc64le is `powerpc64le-linux-gnu`, so this should be `powerpc64le-linux-gnu-objcopy`. This is defined in the GCC source, and propagated pretty much everywhere we would use GNU tooling (like binutils here)

Reply:
Changed